docs/release-notes/release-notes-0.18.0.md

```diff
@@ -418,6 +418,9 @@ bitcoin peers' feefilter values into account](https://github.com/lightningnetwor
 * [Fix a bug](https://github.com/lightningnetwork/lnd/pull/8595) when retrying
   SQL InvoiceDB transactions due to database errors.
 
+* [Turn `sqldb` into a separate
+  package](https://github.com/lightningnetwork/lnd/pull/8603).
```

Comment:
nit: it's actually a Go "module" now, it already was a package before.

Reply:
Fixed!